app/views/courses/upload_roster.html.erb

```diff
@@ -1,5 +1,11 @@
+<% if @roster_error %>
+<div class="error" id="flash_error">
```

Comment:
nit: visually maybe make it so that when you hover over a row it doesn't highlight? I feel like it isn't really necessary

Reply:
Discussed offline as won't fix